packages/errors/src/index.ts

```diff
@@ -3,3 +3,15 @@ export class GitError extends Error {
     super(`Error: Git - ${message}, exit code: ${exitCode}`);
   }
 }
+
+export class ValidationError extends Error {
+  constructor(message: string) {
+    super(`Validation Error: ${message}`);
```

Comment:
I'd rather have it pass the message directly because either one of two things will happen with the error, the error will be handled specially and some code can make it say that it's a validation error or the error will be logged and the name of error will be shown.

```suggestion
    super(message);
```

Reply:
Hey @mitchellhamilton, 

If I follow your recommendation correctly it is because if we catch the error and log it, We don't want say it is validation error right?

Example:

```
try {
  throw new ValidationError('Something something')
} catch (err) {
  /**
  * Will logging `Validation Error: Something something` be useful or `Something 
  * something`
  **/
  console.log(err);
}
```